modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-azure-customizations.adoc
+// * installing/installing_aws/installing-azure-user-infra.adoc
+
+ifeval::["{context}" == "installing-azure-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-azure-user-infra"]
+:upi:
+endif::[]
+
+//mpytlak: The procedure differs depending on whether this module is used in an IPI or UPI assembly.
+
+:_content-type: PROCEDURE
+[id="installation-azure-marketplace-subscribe_{context}"]
+= Selecting an Azure Marketplace image
+If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes. When obtaining your image, consider the following:
+
+* While the images are the same, the Azure Marketplace publisher is different depending on your region. If you are located in North America, specify `redhat` as the publisher. If you are located in EMEA, specify `redhat-limited` as the publisher.
+* The offer includes a `rh-ocp-worker` SKU and a `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker` SKU represents a Hyper-V generation version 2 VM image. The default instance types used in {product-title} are version 2 compatible. If you are going to use an instance type that is only version 1 compatible, use the image associated with the `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker-gen1` SKU represents a Hyper-V version 1 VM image.
+
+.Prerequisites
+
+* You have installed the Azure CLI client `(az)`.
+* Your Azure account is entitled for the offer and you have logged into this account with the Azure CLI client.
+
+.Procedure
+
+. Display all of the available {product-title} images by running one of the following commands:
+** North America:
++
+--
+[source,terminal]
+----
+$  az vm image list --all --offer rh-ocp-worker --publisher redhat -o table
+----
+.Example output
+[source,terminal]
+----
+Offer          Publisher       Sku                 Urn                                                             Version
+-------------  --------------  ------------------  --------------------------------------------------------------  --------------
+rh-ocp-worker  RedHat          rh-ocp-worker       RedHat:rh-ocp-worker:rh-ocpworker:4.8.2021122100               4.8.2021122100
+rh-ocp-worker  RedHat          rh-ocp-worker-gen1  RedHat:rh-ocp-worker:rh-ocp-worker-gen1:4.8.2021122100          4.8.2021122100
```

Comment:
Looks like the second row, last column value isn't flush with the column heading.

Reply:
Thanks. Fixed.